library/x509_create.c

```diff
@@ -95,6 +95,7 @@ int mbedtls_x509_string_to_names( mbedtls_asn1_named_data **head, const char *na
     int in_tag = 1;
     char data[MBEDTLS_X509_MAX_DN_NAME_SIZE];
     char *d = data;
+    int in_quot = 0;
```

Comment:
Why not change it to a boolean if we're only doing logic operations on it?

Reply:
Do we have defined a boolean type somewhere?

Reply:
We don't have a boolean type, and an `int` is usually used.